lib/adal.js

```diff
@@ -810,6 +843,9 @@ var AuthenticationContext = (function () {
                 this.verbose('Window is redirecting');
                 callback = this.callback;
             }
+            callback = this.callback;
+            if (callback)
+                callback(this._getItem(this.CONSTANTS.STORAGE.ERROR_DESCRIPTION), requestInfo.parameters[this.CONSTANTS.ACCESS_TOKEN] || requestInfo.parameters[this.CONSTANTS.ID_TOKEN]);
             window.location = this._getItem(this.CONSTANTS.STORAGE.LOGIN_REQUEST);
```

Comment:
if popUp is set to true, we should not redirect the main app to login start page, as it is already on that page.